src/components/List/AddCard.js

```diff
@@ -0,0 +1,198 @@
+          id: user.userId,
+        }));
+      setAllUserInGroup(groupUserNames);
+      setIsLoading(false);
+    };
+    getUserInGroup();
+  }, [selectedGroup]);
+
+  const handleDropDown = (e) => {
+    setSelectedGroup(e.target.value);
+  };
+
+  const handleSelectUser = (e) => {
+    setSelectedUser(e.target.value);
+  };
+
+  const handleSelectDate = (e) => {
+    setSelectedDate(e.target.value);
+  };
+
+  const handleTitel = (e) => {
+    setTitel(e.target.value);
+  };
+
+  const handleSave = async (e) => {
+    if (title && selectedDate && selectedGroup && selectedUser) {
+      await addListEntry({
+        title,
+        date: selectedDate,
+        groupID: selectedGroup,
+        assignedTo: selectedUser,
+        list,
+      }); // Liste abstrahieren
+      setSnackbarContent({
+        message: "Dein Eintrag wurde erfolgreich erstellt.",
+        status: "success",
+        open: true,
+      });
+      triggerUpdate();
+      close();
+      setSelectedGroup("");
+      setTitel("");
+      setSelectedDate("");
+      setSelectedUser("");
+    } else {
+      setSnackbarContent({
+        message: "Fülle bitte alle Felder aus!",
+        status: "error",
+        open: true,
+      });
+    }
+  };
+
+  return (
+    <>
+      <Snackbar snackbarContent={snackbarContent} setSnackbarContent={setSnackbarContent} />
+      <Dialog open={open} onClose={close} className={classes.dialog}>
+        <DialogTitle id="filme-serien-hinzufügen" className={classes.dialogTitle} variant="h1">
+          {cardTitle} hinzufügen
+        </DialogTitle>
+        <DialogContent dividers>
+          <form className={classes.root} noValidate autoComplete="off">
+            <TextField
+              id="movie title"
+              label="Titel"
+              onChange={handleTitel}
+              value={title}
+              style={{ marginBottom: 10 }}
+              className={classes.textField}
+            />
+            <TextField
+              id="date"
+              onChange={handleSelectDate}
+              label="Frist"
+              type="date"
+              value={selectedDate}
+              className={classes.textField}
+              InputLabelProps={{
+                shrink: true,
+              }}
+            />
+            <FormControl className={classes.formControl}>
+              <InputLabel htmlFor="selectGroup">Gruppe</InputLabel>
+              <Select native value={selectedGroup} onChange={handleDropDown}>
+                <option aria-label="None" value="" />
+                {!isUserData &&
+                  Object.entries(userData.groups).map((group, idx) => (
+                    <option key={idx} value={group[0]}>
+                      {group[1]}
+                    </option>
+                  ))}
+              </Select>
+            </FormControl>
+            <FormControl className={classes.formControl}>
+              <InputLabel htmlFor="zuständiger">Zuständige*r</InputLabel>
```

Comment:
```suggestion
              <InputLabel htmlFor="zuständiger">Zuständig</InputLabel>
```
Spart uns die Genderformulierung und ist neutraler oder? 😃